tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-pyxis-repository
+  labels:
+    app.kubernetes.io/version: "0.0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >
+    Tekton task to mark all repositories in the mapped snapshot as published in Pyxis.
+    This is currently only intended for the rh-push-to-registry-redhat-io pipeline,
+    so it will convert the values to the ones used for registry.redhat.io releases.
+    E.g. repository "quay.io/redhat-prod/my-product----my-image" will be converted
+    to use registry "registry.access.redhat.com" and repository "my-product/my-image"
+    to identify the right Container Registry object in Pyxis.
+  params:
+    - name: server
+      type: string
+      description: The server type to use. Options are 'production' and 'stage'
+      default: production
+    - name: pyxisSecret
+      type: string
+      description: |
+        The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert
+    - name: snapshotPath
+      description: Path to the JSON file containing the mapped Snapshot spec in the data workspace
+      type: string
+      default: "snapshot.json"
```

Comment:
can we make this `snapshot_spec.json` to be consistent with other tasks? I already think it isn't great we have `mapped_snapshot.json` in some places and `snapshot_spec.json` in others... I don't want to add a third possibility

Reply:
Removing it could be nice - we wouldn't have hit all those issues last week where certain tasks in the pipeline definition didn't define the file parameter so it was not behaving properly. If we had no default, it would've been more obvious. It is sort of nice for the tests that we have defaults, but not sure that makes it worth it

Reply:
I created a follow up for this: https://issues.redhat.com/browse/RHTAPREL-667